android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -51,6 +74,129 @@ private TagDispatcher(Activity activity,
+       */
+      public TagDispatcherBuilder disableSounds(boolean disableSounds){
+          this.nestedDisableSounds = disableSounds;
+          return this;
+      }
+
+      /**
+       * Sets if new tags should be dispatched on the UI thread
+       *
+       * @param dispatchOnUiThread  Should new tags be dispatched on UI thread?
+       * @return                    A new instance of TagDispatcherBuilder
+       */
+      public TagDispatcherBuilder dispatchOnUiThread(boolean dispatchOnUiThread){
+          this.nestedDispatchOnUiThread = dispatchOnUiThread;
+          return this;
+      }
+
+      /**
+       * TODO: What does this do?
+       * @param broadcomWorkaround  Should TagDispatcher use the Broadcom workaround?
+       * @return                    A new instance of TagDispatcherBuilder
+       */
+      public TagDispatcherBuilder broadcomWorkaround(boolean broadcomWorkaround){
+          this.nestedBroadcomWorkaround = broadcomWorkaround;
+          return this;
+      }
+
+      /**
+       * Sets if TagDispatcher should use ReaderMode
+       * <p>
+       * TODO: What are the implications of this?
+       *
+       * @param noReaderMode  Don't use ReaderMode?
+       * @return              A new instance of TagDispatcherBuilder
+       */
+      public TagDispatcherBuilder noReaderMode(boolean noReaderMode){
+          this.nestedNoReaderMode = noReaderMode;
+          return this;
+      }
+
+      /**
+       * Sets if TagDispatcher should disable NFC NDEF checking.
+       * <p>
+       * If you are not relying on NDEF or want to run heavy operations we
+       * suggest to disable checking NDEF as keeping it enabled can make it
+       * diffucult to "wake up" secure elements that are stuck in garbage
+       * collection.
+       *
+       * @param disableNdefCheck  Disable NFC NDEF checking?
+       * @return                  A new instance of TagDispatcherBuilder
+       */
+      public TagDispatcherBuilder disableNdefCheck(boolean disableNdefCheck){
+          this.nestedDisableNdefCheck = disableNdefCheck;
+          return this;
+      }
+
+      /**
+       * Build the {@link TagDispatcher}.
+       *
+       * @return A new TagDispatcher
+       */
+      public TagDispatcher build() {
+        return new TagDispatcher (nestedActivity,
+                                  nestedTagDiscoveredListener,
+                                  nestedHandleUnavailableNfc,
+                                  nestedDisableSounds,
+                                  nestedDispatchOnUiThread,
+                                  nestedBroadcomWorkaround,
+                                  nestedNoReaderMode,
+                                  nestedDisableNdefCheck);
+      }
+    }
+
+    /**
+    * @deprecated  As of Nordpol 0.2, replaced by
+    *              {@link TagDispatcherBuilder}.
+    *              Will be removed in v0.3.0 or v1.0.0 (whichever comes first)
+    */
```

Comment:
Please also add the `@Deprecated` java annotation as well right before the method declaration.